docs/source/initialsetup/compilingmame.rst

```diff
@@ -71,7 +71,7 @@ building MAME on a 64-bit system.  Instructions may need to be adjusted for
   window management, audio/video output, font rendering, etc.  If you want to
   use the portable SDL (Simple DirectMedia Layer) interfaces instead, you can
   add **OSD=sdl** to the make options.  The main emulator binary will have an
-  ``sdl`` prefix prepended (e.g. ``sdlmame64.exe`` or ``sdlmame.exe``).  You
+  ``sdl`` prefix prepended (e.g. ``sdlmame.exe`` or ``sdlmame.exe``).  You
```

Comment:
I think  you forgot to read this one – sdlmame or sdlmame.

Reply:
Yeah. Oops. Fixed.